### PR TITLE
feat: added birthdays() with custom period

### PR DESCRIPTION
**Pull request by Sysqwerty**
image

### CODE REVIEW

File: bot_cli/address_book_classes.py [hunk 1]
```diff
@@ -146,10 +146,9 @@ def custom_serializer(obj):
 
             json.dump(self.data, file, default=custom_serializer, indent=4)
 
-    def get_birthdays_per_week(self, period: int = 7) -> dict | None:
-        users_dictionary_per_weekday = defaultdict(list)
+    def get_birthdays_per_period(self, period: int = 7) -> dict | None:
+        users_to_celebrate = defaultdict(list)
```
Comment (LadaM) [inline bot_cli/address_book_classes.py:150]: `upcoming_birthdays` might be a better name

File: bot_cli/commands.py [hunk 2]
```diff
@@ -227,13 +226,17 @@ def show_all_contacts():
...
+        print_info("\n".join(result))
     else:
-        print(Fore.LIGHTBLUE_EX + "No contacts have been added yet")
+        print_info("No contacts have been added yet")
 
 
 @max_period_error
 def birthdays(args):
+    """
+    Show all birthdays in defined period
```
Comment (LadaM) [inline bot_cli/commands.py:237]: ...upcoming birthdays...
default period is 7 days

File: bot_cli/address_book_classes.py (hunk 1 above)
Comment (Sysqwerty) [reply bot_cli/address_book_classes.py:150]: done

File: bot_cli/commands.py (hunk 2 above)
Comment (Sysqwerty) [reply bot_cli/commands.py:237]: done

**Review by LadaM — APPROVED** [review]

**Pull request merged**